### PR TITLE
Fix spelling

### PR DESCRIPTION
**Pull request by jbampton**
### Description

This PR fixes some typos

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [X] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Feature/Enhancement Scale

- [ ] Major
- [X] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [ ] Minor
- [X] Trivial

### CODE REVIEW

**Review by DaanHoogland — COMMENTED** [review]
clgtm

File: services/secondary-storage/server/src/main/java/org/apache/cloudstack/storage/resource/NfsSecondaryStorageResource.java [hunk 1]
```diff
@@ -1139,7 +1139,7 @@ protected File downloadFromUrlToNfs(String url, NfsTO nfs, String path, String n
             }
             return new File(destFile.getAbsolutePath());
         } catch (IOException e) {
-            s_logger.debug("Faild to get url:" + url + ", due to " + e.toString());
+            s_logger.debug("Failed to get url:" + url + ", due to " + e.toString());
```
Comment (SadiJr) [inline services/secondary-storage/server/src/main/java/org/apache/cloudstack/storage/resource/NfsSecondaryStorageResource.java:1142]: LGTM, just a small suggestion:

```suggestion
            s_logger.debug("Failed to get url: " + url + ", due to " + e.toString());
```

**Review by SadiJr — APPROVED** [review]
CLGTM

**Review by rohityadavcloud — APPROVED** [review]

**Comment by rohityadavcloud** [comment]
@blueorangutan package

**Pull request merged**